src/main/kotlin/frc/chargers/hardware/sensors/DriverCamManager.kt

```diff
@@ -0,0 +1,27 @@
+package frc.chargers.hardware.sensors
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+import edu.wpi.first.networktables.NetworkTable
+import edu.wpi.first.networktables.NetworkTableEntry
+import edu.wpi.first.networktables.NetworkTableInstance
+import edu.wpi.first.cameraserver.CameraServer
+
+public class DriverCamManager(
+    public val totalDriverCams: Int = 0,
+    public val defaultResWidth: Int = 720,
+    public val defaultResHeight: Int = 1280){
+    internal var driverCameras = Array(totalDriverCams){i -> CameraServer.startAutomaticCapture(i)}.toList()
```

Comment:
Why internal and not private?

Reply:
I might want to use it in functions right? private would prevent me from doing that